### ISSUE TITLE
Constants as array option

### ISSUE THREAD

**Issue by berkayunal** [klesun/deep-assoc-completion#92]
Hi,

I am trying to use constants in return type but it does not show autocomplete for constants. 

```
/**
* @return array = [
*      "sort_by" => [static::SORT_TYPE_DEFAULT,"type","modified_time"][$enum],
* ]
*/
```

I have also tried this way but still no luck

```
/**
* @return array = [
*      "sort_by" => [BrandModel::SORT_TYPE_DEFAULT,"type","modified_time"][$enum],
* ]
*/
```

How to achieve this? Is it possible?

Thanks.

**Comment by klesun** [comment]
Hm, it's strange that `BrandModel::SORT_TYPE_DEFAULT` does not work. Does your file look like this?
![image](https://user-images.githubusercontent.com/5202330/58695411-dd1b8200-8384-11e9-949f-2c7f9ec2f4be.png)
```php
<?php
class BrandModel
{
    const SORT_TYPE_DEFAULT = 'default';
    
    /** 
     * @return array = [
     *     "sort_by" => [BrandModel::SORT_TYPE_DEFAULT, 'type', 'modified_time'][$enum],
     * ]
     */
    public function doStuff() {
    }
}

(new BrandModel())->doStuff()['sort_by'] === '';
```

The `self::CONSTANT_NAME` is indeed not supported yet - I'll add in next release.

If you provide me with full code of minimal test case of `BrandModel::SORT_TYPE_DEFAULT`, I could debug it as well...

**Comment by klesun** [comment]
Sorry for the delay.
Released in [2019.06.13.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/64014)
